srcpkgs/pipewire/patches/fix-neon-detection.patch

```diff
@@ -0,0 +1,52 @@
+diff --git meson.build meson.build
```

Comment:
if this is really the revert of an upstream commit, be so kind and note that here, ideally linking upstream issue discussion (if any) for later reference